src/inspector_io.cc

```diff
@@ -244,7 +245,9 @@ void InspectorIo::WorkerRunIO() {
   std::string script_path;
   if (!script_name_.empty()) {
     uv_fs_t req;
-    if (0 == uv_fs_realpath(&loop, &req, script_name_.c_str(), nullptr))
+    req.ptr = nullptr;
+    err = uv_fs_realpath(&loop, &req, script_name_.c_str(), nullptr);
+    if (0 == err && req.ptr != nullptr)
```

Comment:
The nullptr check shouldn't be necessary but perhaps coverity isn't smart enough to figure that out...  I'd turn it into a `CHECK_NE(req.ptr, nullptr)`.